### PR TITLE
Add custom field for faceted search

### PR DESCRIPTION
**Pull request by kewitham**
This PR adds a custom meta search field for search.gov to scrape so we will be able to display meta descriptions as the search result descriptions instead of snippets when they are available.

### CODE REVIEW

**Comment by c2nelson** [comment]
@kewitham In this preview the news item is back.  Am I looking at an older view?  I removed it a while ago.

**Comment by kewitham** [comment]
> @kewitham In this preview the news item is back. Am I looking at an older view? I removed it a while ago.

@c2nelson on the home page? I'm not seeing that. Maybe you checked the federalist link before my updates were pushed.

File: assets/js/netlify/makeHTMLFromBodyContent.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ function makeHTMLFromBodyContent(bodyContent, variables, imageData) {
   let content = bodyContent?.replaceAll(/<hr>/g, "***")
       .replaceAll("{{'", '')
       .replaceAll("' | relative_url}}", '')
+      .replaceAll("'| relative_url}}", '')
+      .replaceAll("'|relative_url}}", '')
```
Comment (okkays) [inline assets/js/netlify/makeHTMLFromBodyContent.js:12]: You could use a regex for this, too - that would also account for the ability to have spaces before the `}}`, and to have an arbitrary number of spaces:

https://regex101.com/r/EjI2lY/2

```suggestion
      .replaceAll(/\s*\|\s*relative_url\s*}}/, '')
```

**Review by okkays — APPROVED** [review]
LGTM, one comment

**Pull request merged**